src/classes/UTIL_Describe.cls

```diff
@@ -560,11 +613,15 @@ public class UTIL_Describe {
     */
     public static String getHhAccRecTypeID() {
         String recTypeId = getRecTypesMapByDevName('Account').get('HH_Account');
+
         if(String.isBlank(recTypeId) && Test.isRunningTest()) {
+
             if(getRecTypesMapByDevName('Account').values().size() > 1) {
                 return getRecTypesMapByDevName('Account').values()[1];
             }
-            else return null;
+            else {
```

Comment:
@skosgi1 would you mind bringing this "else" up to the line above it so we have "cuddled" elses?  e.g. } else {